### PR TITLE
Feature landingpage

### PR DESCRIPTION
**Pull request by matthiaswahr**
This is a wild one. Thanks Gabi :-)

### CODE REVIEW

File: client/src/components/Form.js [hunk 1]
```diff
@@ -128,41 +130,18 @@ const Form = styled.form`
...
-  border-radius: 0.4rem;
-  border: none;
   cursor: pointer;
+  font-size: 1.3em;
+  padding: 5px 12px;
+  font-family: Roboto, sans-serif;
+  font-weight: 300;
+  color: teal;
+  border: 1px solid silver;
+  background-image: linear-gradient(to top, gainsboro 0%, white 90%);
```
Comment (GabrieleGrimme) [inline client/src/components/Form.js:143]: You could better use variables in the globalstyle to cope with color. You could change it easier then for the whole.

File: client/src/components/Card.js [hunk 2]
```diff
@@ -26,10 +28,6 @@ export default function VaccinationCard({
     );
   }
 
-  function handleClick(clickedVacc) {
-    onSetEdited(clickedVacc);
-  }
-
   return (
```
Comment (GabrieleGrimme) [inline client/src/components/Card.js:31]: Please check the date component. It shows in deploment the wrong date when it is recalled.

File: client/src/App.js [hunk 3]
```diff
@@ -4,8 +4,6 @@ import { Switch, Route } from 'react-router-dom';
 import LandingPage from './pages/LandingPage';
```
Comment (GabrieleGrimme) [inline client/src/App.js:4]: I recommend using 'display: grid; place-items:center' to get the components in "shape" that they are shown in the middle of the screen.

Comment (matthiaswahr) [reply client/src/App.js:4]: Thanks for your review.

**Pull request merged**